engine/src/main/java/org/hawkular/alerts/engine/impl/ispn/IspnDefinitionsServiceImpl.java

```diff
@@ -138,7 +140,7 @@ public void addActionDefinition(String tenantId, ActionDefinition actionDefiniti
         if (found != null) {
             throw new FoundException(pk);
         }
-        backend.put(pk(actionDefinition), new IspnActionDefinition(actionDefinition));
+        backend.getAdvancedCache().withFlags(IGNORE_RETURN_VALUES).put(pk(actionDefinition), new IspnActionDefinition(actionDefinition));
```

Comment:
Would it make sense to turn `backend` into a new class `SuperCache` (that inherits from `Cache`) which then gets a method `putNR()`, which internally supplies the flag? This way you would not need the `.getAC().withFlags()` all over the place

Reply:
If we wanted to keep using `backend`, yes definitely. Now, with policies-history coming soon (hopefully!) I don't know if that's worth it.

Reply:
@pilhuhn Can we merge the current version or would you prefer the improved one?

Reply:
Whatever you find right :)

Reply:
Let's merge this then, I'd rather stay focused on the main part 😃